lang/src/org/partiql/lang/eval/visitors/PartiqlAstSanityValidator.kt

```diff
@@ -67,25 +73,12 @@ class PartiqlAstSanityValidator : PartiqlAst.Visitor() {
         }
     }
 
-    private fun validateDecimalOrNumericType(precision: Long?, scale: Long?, metas: MetaContainer) {
-        if (scale != null && precision != null && compileOptions.typedOpBehavior == TypedOpBehavior.HONOR_PARAMETERS) {
-            if (scale !in 0..precision) {
-                err(
-                    "Scale $scale should be between 0 and precision $precision",
-                    errorCode = ErrorCode.SEMANTIC_INVALID_DECIMAL_ARGUMENTS,
-                    errorContext = errorContextFrom(metas),
-                    internal = false
-                )
-            }
-        }
-    }
-
```

Comment:
This is validation of `DECIMAL` type parameters, and we are moving it to the `FloatType` defined  legacy plugin package.